apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -154,16 +154,35 @@ module.exports = class Filepanel extends ViewPlugin {
     })
   }
 
+  saveRecent (workspace) {
+    if (!localStorage.getItem('recentWorkspaces')) {
+      localStorage.setItem('recentWorkspaces', JSON.stringify('"first": "", "second": "", "third": ""'))
```

Comment:
why it's not just an array.... ?

Reply:
i don't know. it feels like manipulation would be easier like that